docs/waveform_plugin.rst

```diff
@@ -0,0 +1,91 @@
+
+Ther are two kinds of models you can make. In the example below, we
+make a time-domain model. You can also make a freuqency-domain model. The only
+difference is that your function should return pycbc FrequencySeries and
+the required sample step option is 'delta_f' instead of 'delta_t'. 
+
+Each waveform generation function must take only keyword arguments, and
+should be able to take an arbitrary number of them. You may add new parameters
+as you like. These will be automatically useable by PyCBC Inference and
+other pycbc codes.
+
+Each waveform model must have an associate 'approximant' name, which identifies
+the model and distinguishes it from any other. If the name has already been
+used, you may need to select a different name. 
+
+.. plot:: ../examples/waveform/add_waveform.py
+   :include-source:
+
+=================================================
+Creating a plugin for PyCBC
+=================================================
+
+To make a waveform model universally available to PyCBC so it can be called
+from PyCBC Inference, or the pycbc-based searched codes, you can create
+a plugin package which advertises your model. PyCBC will automatically
+detects your package and makes your waveform model available for use.
+
+The steps are:
+
+ * Create a waveform model just like as in the above example
+ * Create a python package for your module
+ * In your packages setup.py advertise that it contains a PyCBC compatible
+   waveform model in it's 'entry_points' option. 
+  
+The setup.py would like the following, the key addition being the 'entry_points'
+parameter passed to setup.py. 
+
+.. code-block:: python
+
+    setup (
+        name = 'pycbc-revchirp',
+        version = VERSION,
+        description = 'An example waveform plugin for PyCBC',
+        long_description = open('descr.rst').read(),
+        author = 'The PyCBC team',
+        author_email = 'alex.nitz@gmail.org',
+        url = 'http://www.pycbc.org/',
+        download_url = 'https://github.com/gwastro/revchirp/tarball/v%s' % VERSION,
+        keywords = ['pycbc', 'signal processing', 'gravitational waves'],
+        install_requires = ['pycbc'],
+        py_modules = ['revchirp'],
+        entry_points = {"pycbc.waveform.td":"revchirp = revchirp:reverse_chirp_td",
+                        "pycbc.waveform.fd":"revchirp = revchirp:reverse_chirp_fd"},
+        classifiers=[
+            'Programming Language :: Python',
+            'Programming Language :: Python :: 2.7',
+            'Programming Language :: Python :: 3.6',
+            'Intended Audience :: Science/Research',
+            'Natural Language :: English',
+            'Topic :: Scientific/Engineering',
+            'Topic :: Scientific/Engineering :: Astronomy',
+            'Topic :: Scientific/Engineering :: Physics',
+            'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        ],
+    )
+
+The format is "capability":"approximant_name = module_path:function_name".
```

Comment:
Change to, "The format for the `entry_points` parameter is `"capability":"approximant_name = module_path:function_name"`." to make it clear you're referring to the `entry_points` (and use backticks for the format).